config/labels/analyzers/clangsa.json

```diff
@@ -1077,6 +1076,7 @@
     ],
     "unix.cstring.NullArg": [
       "doc_url:https://clang.llvm.org/docs/analyzer/checkers.html#unix-cstring-nullarg-c",
+      "guideline:sei-cert",
```

Comment:
This is the only behavioural change, but if someone uses `analyse --disable default --enable guideline:sei-cert` (or however it should be specified...) then they will start seeing this checker, which changes the potentially reported set of bugs.